core/src/main/java/com/sequenceiq/cloudbreak/service/externaldatabase/AzureDatabaseServerParameterDecorator.java

```diff
@@ -11,17 +11,30 @@
 @Component
 public class AzureDatabaseServerParameterDecorator implements DatabaseServerParameterDecorator {
 
-    @Value("${cb.azure.externaldatabase.retentionperiod:7}")
-    private int retentionPeriod;
+    @Value("${cb.azure.externaldatabase.ha.retentionperiod}")
+    private int retentionPeriodHa;
 
-    @Value("${cb.azure.externaldatabase.geoRedundantBackup:true}")
-    private Boolean geoRedundantBackup;
+    @Value("${cb.azure.externaldatabase.ha.georedundantbackup}")
+    private Boolean geoRedundantBackupHa;
+
+    @Value("${cb.azure.externaldatabase.nonha.retentionperiod}")
+    private int retentionPeriodNotHa;
```

Comment:
Nit: nonHa vs notHa usage at different places.

Reply:
Thank you, changed.